scrapy_po/utils.py

```diff
@@ -26,19 +26,23 @@ def __init__(self, url, request=None):
 
 def is_callback_using_response(callback: Callable):
     """Check whether the request's callback method is going to use response."""
-    spec = inspect.getfullargspec(callback)
-    try:
-        arg_name = spec.args[1]  # first index is self, second is response
-    except IndexError:
+    if getattr(callback, '__scrapy_po_callback', False) is True:
```

Comment:
This is a new check.

Reply:
Could you please move the string to a constant (_CALLBACK_FOR_MARKER or something like that)?